ciao-controller/legacy_api.go

```diff
@@ -83,211 +75,75 @@ func (h legacyAPIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-// @Failure 400 {object} HTTPReturnErrorCode "The response contains the corresponding message and 40x corresponding code."
-// @Failure 500 {object} HTTPReturnErrorCode "The response contains the corresponding message and 50x corresponding code."
-// @Router /v2.1/cncis [get]
-// @Resource /v2.1/cncis
 func legacyListCNCIs(c *controller, w http.ResponseWriter, r *http.Request) (APIResponse, error) {
 	return listCNCIs(c, w, r)
 }
 
-// @Title legacyListCNCIDetails
-// @Description List details of a CNCI agent.
-// @Accept  json
-// @Success 200 {array} types.CiaoCNCIs "Returns details of a CNCI agent as InstanceId, TenantID, IPv4 and subnets."
-// @Failure 400 {object} HTTPReturnErrorCode "The response contains the corresponding message and 40x corresponding code."
-// @Failure 500 {object} HTTPReturnErrorCode "The response contains the corresponding message and 50x corresponding code."
-// @Router /v2.1/cncis/{cnci}/detail [get]
-// @Resource /v2.1/cncis
 func legacyListCNCIDetails(c *controller, w http.ResponseWriter, r *http.Request) (APIResponse, error) {
 	return listCNCIDetails(c, w, r)
 }
 
-// @Title legacyListTraces
-// @Description List all Traces.
-// @Accept  json
-// @Success 200 {array} types.CiaoTracesSummary "Returns a summary of each trace in the system."
-// @Failure 400 {object} HTTPReturnErrorCode "The response contains the corresponding message and 40x corresponding code."
-// @Failure 500 {object} HTTPReturnErrorCode "The response contains the corresponding message and 50x corresponding code."
-// @Router /v2.1/traces [get]
-// @Resource /v2.1/traces
 func legacyListTraces(c *controller, w http.ResponseWriter, r *http.Request) (APIResponse, error) {
 	return listTraces(c, w, r)
 }
 
-// @Title legacyListEvents
-// @Description List all Events.
-// @Accept  json
-// @Success 200 {array} types.CiaoEvent "Returns all events from the log system."
-// @Failure 400 {object} HTTPReturnErrorCode "The response contains the corresponding message and 40x corresponding code."
-// @Failure 500 {object} HTTPReturnErrorCode "The response contains the corresponding message and 50x corresponding code."
-// @Router /v2.1/events [get]
-// @Resource /v2.1/events
 func legacyListEvents(c *controller, w http.ResponseWriter, r *http.Request) (APIResponse, error) {
 	return listEvents(c, w, r)
 }
 
-// @Title legacyListTenantEvents
-// @Description List Events.
-// @Accept  json
-// @Success 200 {array} types.CiaoEvent "Returns the events of a tenant from the log system."
-// @Failure 400 {object} HTTPReturnErrorCode "The response contains the corresponding message and 40x corresponding code."
-// @Failure 500 {object} HTTPReturnErrorCode "The response contains the corresponding message and 50x corresponding code."
-// @Router /v2.1/{tenant}/events [get]
-// @Resource /v2.1/events
 // listTenantEvents is created with the only purpose of API documentation for method
 // /v2.1/{tenant}/events
```

Comment:
I checked this, this is needed for ciao-cli event list (verified by deleted and observing the ciao-cli fails.)